<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>285</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Сервер:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="text">
        <string>127.0.0.1:28080</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Соединить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLed" name="labelStatus">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>15</width>
         <height>15</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>:/status/images/icon_grn.ico</pixmap>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>491</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Запрос</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionDirs"/>
    <addaction name="actionFiles"/>
    <addaction name="actionFileUnfo"/>
    <addaction name="actionRead"/>
    <addaction name="actionTempCopy"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>О сервере</string>
   </property>
  </action>
  <action name="actionDirs">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Список папок</string>
   </property>
  </action>
  <action name="actionFiles">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Список файлов</string>
   </property>
  </action>
  <action name="actionFileUnfo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>О файле</string>
   </property>
  </action>
  <action name="actionRead">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Чтение файла</string>
   </property>
  </action>
  <action name="actionTempCopy">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Временная копия</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QLed</class>
   <extends>QLabel</extends>
   <header>qled.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
